compiler/main/driver.cpp

```diff
@@ -506,6 +506,21 @@ static void setupChplHome(const char* argv0) {
   }
 }
 
+#ifndef HAVE_LLVM
+
+// If the compiler was built without LLVM and CHPL_LLVM is not set in
+// the environment, set it to 'none' in order to avoid having the
+// chplenv scripts potentially infer it to be 'system' or 'bundled',
+// which can only result in an error.
+//
+static void setupChplLLVM(void) {
+  // set CHPL_LLVM to 'none' if it isn't already set
+  if (setenv("CHPL_LLVM", "none", 0) != 0) {
+    INT_FATAL("Problem setting CHPL_LLVM");
+  }
```

Comment:
I was expecting this to call `getenv` first to see if CHPL_LLVM is set, and if not, do the `setenv` *and print a warning*.